src/Apps/Auction/__tests__/routes.test.ts

```diff
@@ -120,6 +120,21 @@ describe("Auction/routes", () => {
       )
     })
 
+    it("redirects to the login (plus redirect_uri of sale artwork bid page) if user is not signed in", async () => {
+      const fixture: BidQueryResponse = mockConfirmBidResolver()
+      ;(fixture as any).me = null
```

Comment:
instead of casting to `any`, thoughts on editing the type of the `BidQueryResponse` to permit a `null` at the `me` attribute? I think we can use this as an opportunity to make our test data more realistic without breaking type safety.

Reply:
Pardon the nooby question, what's the deal with the  `;(...)` syntax? I have been struggling to google around for it.

Reply:
Without it, prettier gets confused and tries to join it with the line above.